sgl-kernel/csrc/moe/moe_reorder_kernel.cu

```diff
@@ -0,0 +1,64 @@
+#include <cuda_runtime.h>
```

Comment:
It looks like you do not add the cu file and cc file into `sgl-kernel/CMakeLists.txt`.